### PR TITLE
Update tfjs version to 1.0.2 for SketchRNN

### PR DESCRIPTION
**Pull request by joeyklee**
Hello,

## Thanks
Thanks for all the wonderful work with magenta-js! It's a wonderful library. 

## Intentions
Our team over at https://ml5js.org/ have been working on updating ml5 to tensorflow 1.0.2 and realized that magenta's SketchRNN is still using tfjs 0.13.0. In an effort to sync up all of our dependencies, we'd love to updated SketchRNN to tfjs 1.0+

## The update
In order to update SketchRNN I followed along how you all set up the webpack build for the magenta-js/music here: https://github.com/tensorflow/magenta-js/tree/master/music. It seemed like there was already a discussion forming here: https://github.com/tensorflow/magenta-js/issues/296.

I pretty much copied the setup and made a few adjustments here and there. All of the demos are working so I assume things are running alright so far.

If you'd be willing to make these updates available for your wider magenta community, that would be super! Many thanks! 

Screen Shot 2019-05-13 at 18 23 32

### CODE REVIEW

**Comment by adarob** [comment]
Thanks for doing this! @notwaldorf can you review?

**Comment by notwaldorf** [comment]
YES!!! Thank you SO much for this PR! 😍😍😍
I'll look at it today -- could you make sure you sign the CLA, though? Can't merge it until you do.

Again, thank you so so much for doing the work!

**Comment by joeyklee** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here (e.g. `I signed it!`) and we'll verify it.
> #### What to do if you already signed the CLA
> ##### Individual signers
> 
>     * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> 
> ##### Corporate signers
> 
>     * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google.com/docs/cla/#troubleshoot)).
> 
>     * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
>     * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Fmagenta-js%2Fpull%2F310) for more info**.

I signed it

🌈

(I might have to change my email address for the CLA. I will check now)

**Comment by notwaldorf** [comment]
Actually, one more thing: would you mind _not_ committing the docs files? It makes the review super hard since there's so much noise. (We usually release all the docs in a separate PR, after deploying to npm)

**Comment by joeyklee** [comment]
> Actually, one more thing: would you mind _not_ committing the docs files? It makes the review super hard since there's so much noise. (We usually release all the docs in a separate PR, after deploying to npm)

+ Ooh! Whoops. This makes sense. I will remove those now. Just a moooment.

**Comment by notwaldorf** [comment]
Actually, one more thing: would you mind _not_ committing the docs files? It makes the review super hard since there's so much noise. (We usually release all the docs in a separate PR, after deploying to npm)

**Comment by joeyklee** [comment]
> Actually, one more thing: would you mind _not_ committing the docs files? It makes the review super hard since there's so much noise. (We usually release all the docs in a separate PR, after deploying to npm)

Hi there! I just reset the docs files and recommitted them from `master`. Is that ok? https://github.com/tensorflow/magenta-js/pull/310/commits/1a5bb649c576da1f624fed62059eb3274c20b022

File: sketch/package.json [hunk 1]
```diff
@@ -7,34 +7,50 @@
   "jsdelivr": "dist/magentasketch.js",
   "unpkg": "dist/magentasketch.js",
   "dependencies": {
-    "@tensorflow/tfjs": "^0.13.3"
+    "@tensorflow/tfjs": "^1.0.2",
+    "@types/node": "^12.0.1",
+    "fs": "0.0.1-security",
```
Comment (notwaldorf) [inline sketch/package.json]: i thiiiink these are still dev dependencies, no?

Comment (joeyklee) [reply sketch/package.json]: Oh! Yes, you're totally right. My mistake! Let me move those out quickly. Pardon the oopsies + thanks for catching this!

File: sketch/scripts/generate-docs.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Copyright 2018 Google Inc. All Rights Reserved.
```
Comment (notwaldorf) [inline sketch/scripts/generate-docs.sh]:
```suggestion
# Copyright 2019 Google Inc. All Rights Reserved.
```

File: sketch/src/lib.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright 2018 Google Inc. All Rights Reserved.
```
Comment (notwaldorf) [inline sketch/src/lib.ts]:
```suggestion
 * Copyright 2019 Google Inc. All Rights Reserved.
```

File: sketch/package.json (hunk 1 above)
Comment (joeyklee) [reply sketch/package.json]: Okie doke. This should now be sorted with : https://github.com/tensorflow/magenta-js/pull/310/commits/9bed2abad2c1a22a8e91bf22ee6eb2b48d0edfcd thank you!

**Comment by notwaldorf** [comment]
Dear whoever reads this in the future: the github suggestion ui used my secondary email, and that's why the CLA is now red. Author is fine.

**Comment by joeyklee** [comment]
@notwaldorf - Thank you so much! It was wonderful to have such quick, responsive, and friendly support on this. Many thanks! ✨

**Comment by notwaldorf** [comment]
Released as 0.2.0.

**Comment by joeyklee** [comment]
> Released as 0.2.0.

Wow! Wonderful. Many many thanks! ✨🌟🎉

**Comment by notwaldorf** [comment]
Yikes, I just noticed you had a stray commit for https://github.com/tensorflow/magenta-js/blob/master/music/package.json, where you removed the `yarn test`. Would you mind reverting that, please?

**Comment by joeyklee** [comment]
> Yikes, I just noticed you had a stray commit for https://github.com/tensorflow/magenta-js/blob/master/music/package.json (https://github.com/tensorflow/magenta-js/pull/310/files#diff-8477647ba193024c727a26963dee47b4R46), where you removed the `yarn test`. Would you mind reverting that, please?

Eep! Sorry about that. This must've gotten mixed up with some of the experiments I had been running. Thank you for catching this + apologies for any damage!

**Comment by notwaldorf** [comment]
np np!

**Comment by joeyklee** [comment]
🙏🤓

**Pull request merged**